ttnn/cpp/ttnn/operations/moreh/moreh_matmul/moreh_matmul.cpp

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_matmul.hpp"
+
+#include "ttnn/operations/moreh/moreh_matmul/device/moreh_matmul_device_operation.hpp"
+
+namespace ttnn::operations::moreh::moreh_matmul {
+Tensor MorehMatmul::invoke(
+    const Tensor& input,
+    const Tensor& other,
+    bool transpose_input,
+    bool transpose_other,
+    const std::optional<Tensor>& output,
+    const std::optional<const Tensor> bias,
+    const std::optional<MemoryConfig>& output_mem_config,
```

Comment:
```suggestion
    const std::optional<MemoryConfig>& output_memory_config,
```
It seems that TT prefers `memory_config` over `mem_config`.